### PR TITLE
Automated cherry pick of #15788

### PR DESCRIPTION
**Pull request by caesarxuchao**
Cherry pick of #15788 on release-1.1.

### CODE REVIEW

**Comment by caesarxuchao** [comment]
https://htmlpreview.github.io/?https://raw.githubusercontent.com/caesarxuchao/kubernetes/automated-cherry-pick-of-%2315788-upstream-release-1.1/docs/api-reference/extensions/v1beta1/definitions.html
https://htmlpreview.github.io/?https://raw.githubusercontent.com/caesarxuchao/kubernetes/automated-cherry-pick-of-%2315788-upstream-release-1.1/docs/api-reference/extensions/v1beta1/operations.html
https://htmlpreview.github.io/?https://raw.githubusercontent.com/caesarxuchao/kubernetes/automated-cherry-pick-of-%2315788-upstream-release-1.1/docs/api-reference/v1/definitions.html
https://htmlpreview.github.io/?https://raw.githubusercontent.com/caesarxuchao/kubernetes/automated-cherry-pick-of-%2315788-upstream-release-1.1/docs/api-reference/v1/operations.html

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Comment by wojtek-t** [comment]
@caesarxuchao - the Shippable failure seems like a real one - can you please take a look?

**Comment by caesarxuchao** [comment]
```
/root/workspace/src/k8s.io/kubernetes/hack/gen-swagger-doc/run-gen-swagger-docs.sh: line 27: docker: command not found
```
I don't understand why this scripts gets to run in Shippable.

File: docs/api-reference/extensions/v1beta1/definitions.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 <!-- needed for gh-pages to render html files when imported -->
 {% include definitions.html %}
 
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/api-reference/extensions/v1beta1/definitions.md).
```
Comment (nikhiljindal) [inline docs/api-reference/extensions/v1beta1/definitions.md]: We dont need this?
Did a script add it?

File: docs/api-reference/extensions/v1beta1/operations.md [hunk 2]
```diff
@@ -5,6 +5,9 @@
 <!-- needed for gh-pages to render html files when imported -->
 {% include operations.html %}
 
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/api-reference/extensions/v1beta1/operations.md).
```
Comment (nikhiljindal) [inline docs/api-reference/extensions/v1beta1/operations.md]: same

File: docs/api-reference/v1/definitions.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/api-reference/v1/definitions.md).
```
Comment (nikhiljindal) [inline docs/api-reference/v1/definitions.md]: This link doesnt work either.
Should be http://releases.k8s.io/release-1.0/docs/api-reference/definitions.md

File: docs/api-reference/extensions/v1beta1/definitions.md (hunk 1 above)
Comment (caesarxuchao) [inline docs/api-reference/extensions/v1beta1/definitions.md]: Perhaps. Let me try to remove it.

**Comment by nikhiljindal** [comment]
From shippable logs it seems like gen-swagger-doc/run-gen-swagger-docs.sh is running because https://github.com/kubernetes/kubernetes/pull/15748 is not cherrypicked?

File: docs/api-reference/extensions/v1beta1/operations.md (hunk 2 above)
Comment (caesarxuchao) [inline docs/api-reference/extensions/v1beta1/operations.md]: according to my commit message, it's added by hack/update-generated-docs.sh. But it's not added back this time after I manually remove it.

File: docs/api-reference/v1/definitions.md (hunk 3 above)
Comment (caesarxuchao) [inline docs/api-reference/v1/definitions.md]: this one will be removed after you run versionized-docs.sh

**Comment by caesarxuchao** [comment]
@nikhiljindal comments addressed, could you add the LGTM label back?

**Comment by caesarxuchao** [comment]
@k8s-bot test this please

File: docs/api-reference/v1/definitions.md (hunk 3 above)
Comment (nikhiljindal) [inline docs/api-reference/v1/definitions.md]: ah yes. But it is broken on HEAD. We should fix that in another PR.

**Comment by nikhiljindal** [comment]
Thanks for the fixes. LGTM.
How are you fixing shippable?

**Comment by nikhiljindal** [comment]
I have requested cherrypick for https://github.com/kubernetes/kubernetes/pull/16060 which should fix shippable for you.

**Comment by caesarxuchao** [comment]
This is blocked by #16060

**Comment by caesarxuchao** [comment]
add back "lgtm" because it's a rebase.

restart testing since #16060 is merged.

**Pull request merged**